README.md

```diff
@@ -1,142 +1,126 @@
-
-## Running Webcert locally on Docker
-
-Pre-requisites: Setup the development environment (Follow instructions in 'develop' on https://github.com/sklintyg/devops)
-
-1. Start the development environment (Follow instructions in 'develop' on https://github.com/sklintyg/devops)
-    * docker-compose up -d (start as daemon)
-    * docker-compose up (will lock console and use it to print container logs)
-2. Start Webcert
-    * gradlew appRun
-3. Start Webcert-frontend (This time using docker-compose.yaml in the frontend repo)
-    * docker-compose up -d (start as daemon)
-    * docker-compose up (will lock console and use it to print container logs)
-4. Navigate to Webcert-frontend in a chromium-browser: https://wc2.wc.localtest.me/welcome
-
-When doing step #3, docker will automatically build an image if it doesn't exist. If you already have an image and want to build a new one follow these steps:
-
-1. Take down the Webcert-frontend
-    * docker-compose down
-2. Rebuild Webcert-frontend
-    * docker-compose build
-3. Start Webcert-frontend
-    * docker-compose up -d (start as daemon)
-    * docker-compose up (will lock console and use it to print container logs)
-4. Navigate to Webcert-frontend in a chromium-browser: https://wc2.wc.localtest.me/welcome
-
-## Deploy Webcert on OpenShift (devtest)
-
-This is a temporary way to deploy Webcert on OpenShift, until the deployment pipeline is ready.
-
-Pre-requisites: Need to have "make" installed. For Windows you can use the following instructions: https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows
-
-1. Open your terminal of choice and log in to OpenShift.
-2. Run `make`
-3. Done! 
-
-A Deployment has been created in tintyg for devtest. It will automatically deploy on image-changes. When running the make command, a new runtime image with Webcert-frontend will built and automatically deployed. 
-
-## Source-to-image
-
-More information can be found here: https://github.com/openshift/source-to-image
-
-To build it locally: `docker build -t s2i-frontend-builder:latest .`
-To build it on openshift: `make`
-
-To use it locally: `s2i build . s2i-frontend-builder:latest webcert-frontend:latest --loglevel 3` (you can skip the --loglevel or even change it to '5' for more loginfo)
-To use it on openshift: Run the pipeline
+  - VS Code
+    - Install plugin `vscode-styled-components`
+    - Install plugin `Prettier - Code formatter`
+    - Enable auto format on save
+      - Press `ctrl` + `shift` + `p`, type `settings` and open `Preferences: Open Settings (JSON)`
+      - Add the following properties
+        ```json
+        {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        }
+        ```
+
+## Run Webcert backend and frontend client
```

Comment:
Should we have app specific information in this instruction? It is convenient and makes it easier the first time someone follows the instructions, but it could also become obsolete if there are changes to Webcert libraries or dependencies.

Reply:
Not sure if I understand you correctly. Which apps are you referring to? And why wouldn't we specify dependencies to frontend in its docs? If dependencies change, so should the docs IMO.
Anyhow, if we remove the instructions relating to Webcert's dependencies we should make sure it's very clear what is needed in Webcert's docs.

Reply:
I'm referring to Webcert. We have instructions in the devops-repo how to run and how to build the different apps. Having specific information in the frontend-repo on how we build Webcert and the libraries it depends on (infra, common, refdata) is redundant, though it can be convenient.

Reply:
Ok, then I vote for including the instructions. Figuring out this part was particularly hard and error prone for me when I started.